service/controller/v2/cloudconfig/cloud_config.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/certs"
-	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_3_3_4"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_3_4_0"
```

Comment:
v2 is going to be released soon. v_3_4_0 is WIP. Please coordinate with @r7vme before merging.

Reply:
We stated before, that k8scloudconfig should be ready for release whenever we need. 
@r7vme any blockers for this?

Reply:
@corest can you please do your changes in v3?

I've already spend two days testing release 1.0.0 and going to activate it tomorrow.